ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/handlers/HelpCommand.java

```diff
@@ -47,13 +55,18 @@ public void run(String... args) {
             String helpFile = PropertiesLoader.getProperty(HELP_FILE);
             String helpContent = FileUtils.readResource(helpFile);
             ballerinaShell.outputInfo(helpContent);
+
         } else {
+            String topic = String.join(" ", Arrays.copyOfRange(args, 0, args.length));
             try {
-                StringBuilder content = new StringBuilder();
-                helpProvider.getTopic(args, content);
-                ballerinaShell.outputInfo(content.toString());
+                ballerinaShell.outputInfo(DESCRIPTION_PREFIX + NEW_LINE +
+                        bbeHelpProvider.getDescription(topic).replaceAll(TAGS, EMPTY_STRING));
+                ballerinaShell.outputInfo(URL_PREFIX + URL +
+                        topic.replaceAll(" ", "-"));
+
             } catch (HelpProviderException e) {
-                ballerinaShell.outputError(e.getMessage());
+                ballerinaShell.outputError(NEW_LINE + "Can Not Find Topic " + topic  + NEW_LINE +
+                        NEW_LINE + "For Available Topics Please Visit " + URL);
```

Comment:
Let's use proper sentence case in messages.

Reply:
Same goes for the `URL_PREFIX` constant

Reply:
Updated in b369170151717249ec60d86bfbeab195a30174d8